packages/calcite-components/conventions/Internationalization.md

```diff
@@ -66,8 +66,10 @@ This pattern enables components to support built-in translations. In order to su
 
 In the future it will likely become necessary to provide string translations for components. An example would be the `aria-label` for the `<calcite-modal>` close button. Initial research looks promising and we could likely implement one of these approaches and set a `lang` for each component.
 
-- <https://medium.com/stencil-tricks/implementing-internationalisation-i18n-with-stencil-5e6559554117> and <https://codesandbox.io/s/43pmx55vo9>
-- <https://github.com/ionic-team/ionic-stencil-conference-app/issues/69>
+<!-- TODO: Get suggestions for what this should be replaced with -->
```

Comment:
We can drop this entire section (`Translated strings`) since it's deprecated and no longer followed.